scripts/build-toolchains.sh

```diff
@@ -141,5 +141,5 @@ cd "$RDIR"
 } > env-$TOOLCHAIN.sh
 
 # create general env.sh
-ln -sf env-$TOOLCHAIN.sh env.sh
+echo "source \$( realpath \$(dirname "\${BASH_SOURCE[0]}") )/env-$TOOLCHAIN.sh" >> env.sh
```

Comment:
I'm not particularly enthused about introducing Bashisms in `env.sh` like `${BASH_SOURCE[0]}`, but it's probably not going to affect the vast majority of users.  As long as the toolchain `env-*.sh` is pure `sh(1)` syntax, I can work around this.